### ISSUE TITLE
When is training code release? Especially the loss part?

### ISSUE THREAD

**Issue by QiaoranC** [acrarshin/RPNet#1]
When is training code release? Especially the loss part?

**Comment by acrarshin** [comment]
This will take a week at worst. Excuse the delay.

**Comment by QiaoranC** [comment]
No rush, i think i may already reproduce the results, good work. The model size is too large, i think it can be go lightly cause the r-peak detect task seems not too hard. I may continue work on that. Thanks.

**Comment by acrarshin** [comment]
Okay. All the best
